### PR TITLE
Fix docker

### PR DESCRIPTION
**Pull request by bkartyas**
I think change of workers solve the problem.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    - secure: "ukgwdYPAaSqjjO81DwAF9/E/vGZxGJflpp8izDWv5Dyfq/9yL5kcGBYJrQb4rWIOI3nXTAbPLPpZ0YmDACEzsbBczEzTtpwuGkHo6x0D3XuA7jfI7CXMVyANgMDIJFGfMAdr4FRrd9K/gXycvoPhvbHWm4glajfiso1Wb5JXPH5KLh2hp3RyHwsGV3w6rnUYQ2x+8lWyoN7uBhMLSvMfoFsu35cPnMXZ4q6F+g10+NQL4/HBbXhHSbiIaFBRkoreWMK1AgOHP7K1HL7RIUZP3h4QCp9K/Xl//wzgq2Co+k15RsGOCdiYpOpYFQbzrmPml4olARpSMQc4SaYSLWQ6W6y9QdksNpIHvk8rRXKCzIOCD4ffLyubocGH0FIzi6Ba6RTc7Sp6XtyZkxdboF8i3apMHMWfbzlrrQwon4zHN0AqvbFuo1pAHDtAeiJyZ6CPMtELgBElm7GMGvr/aiXHrQmIIGtVthX7ue1yQVNllytjdUsG3wUY3r6d+dZ38ZfPpOKVsyrpf0ol2viWK+Bw7rZq+nznCPGLWbqmBjp3qQI2d2TrKkcoYyvkfqvsoL3tKprws/kQpBPr6Nx58Ly0ZOXbCmjGkADNVWY07fyUcTGJzsD8VWlpX//gyTqaGBNs1CQP0LAmyjF3QzAa6GCQoEgr2Ix53BpLW7MXiX/kEzg=" # DOCKER_PASS
```
Comment (zsol) [inline .travis.yml]: Don't think you wanted to commit these secrets

Comment (bkartyas) [inline .travis.yml]: No, I didn't can I somehow completely remove this from here?

Comment (abesto) [inline .travis.yml]: @bkartyas You can remove them from the Git history by squashing the commits. For future reference, there's https://rtyley.github.io/bfg-repo-cleaner/ that can help when a secret has been in a repo for a while, and is harder to manually clean up.

Now since this is a public repository, anything that shows up in a commit should be treated as compromised (to a lesser extent, the same is true for private repositories). Luckily in your case these are not the _actual_ secrets, but only their encrypted form - Travis generates an encryption key per project, which is _not_ committed into the repository, so to abuse these secrets, one would also need access to those secrets. It's standard practice to host these forms of secrets in the `.travis.yml` files of OSS projects.

Long story short: don't panic, remove the secrets and squash the commits (http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html), and you may still want to change those secrets, just to be on the super-safe side.
